CONTRIBUTORS.md

```diff
@@ -165,6 +165,7 @@
 - [tikuf](https://github.com/tikuf/)
 - [Tim Hobbs](https://github.com/timhobbs)
 - [SvenVandenbrande](https://github.com/SvenVandenbrande)
+- [tacohole](https://github.com/tacohole)
```

Comment:
This is the list of past Emby Contributors, the list of Jellyfin Contributors above.